src/main/java/com/hubspot/imap/protocol/command/BaseImapCommand.java

```diff
@@ -21,4 +25,8 @@ public String getPrefix() {
   public ImapCommandType getCommandType() {
     return type;
   }
+
+  String getFlagString(Collection<MessageFlag> flags) {
```

Comment:
why does this live in `BaseImapCommand`?

Reply:
The same logic is used by both the StoreCommand and AppendCommand and I figured more commands would need it to. Should it live in a util instead?